packages/gapi-ocaml/gapi-ocaml.0.3.5/url

```diff
@@ -0,0 +1,2 @@
+archive: "https://github.com/astrada/gapi-ocaml/archive/v0.3.5.tar.gz"
```

Comment:
Hmm, is this a valid way of specifying a URL for the archive? I haven't seen syntax like this before for the archive field.

Reply:
Yes, it fetches tag v0.3.5 (see https://github.com/astrada/gapi-ocaml/releases/tag/v0.3.5, link "Source code (tar.gz)").